src/sos/step_executor.py

```diff
@@ -129,13 +129,6 @@ def get_job(self, all_tasks=False):
         # individual tasks...
         if self.trunk_size == 1 or all_tasks:
             to_be_submitted = self._unsubmitted_tasks
-            [
-                to_be_submitted.extend([x[1]
-                                        for x in slot
-                                        if x[1] is not None])
-                for slot in self._slots
-                if slot
-            ]
```

Comment:
You cannot simply remove a statement.... Change to a for loop if you understand this piece of code.